### PR TITLE
update models

### PR DESCRIPTION
**Pull request by YueErro**
Updated according to https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/11 and https://github.com/AcutronicRobotics/MARA/pull/89.

### CODE REVIEW

File: mara_description/urdf/mara.urdf.xacro [hunk 1]
```diff
@@ -280,39 +280,28 @@
       <selfCollide>True</selfCollide>
     </gazebo>
 
+    <!-- dummy link with the required tags-->
     <link name="${prefix}motor6_link">
       <inertial>
-        <origin rpy="0 0 0" xyz="0.0 0.0 -0.0173"/>
-        <mass value="0.14"/>
-        <inertia ixx="9.22e-05" ixy="6.35e-7" ixz="2.00e-11" iyy="9.73e-5" iyz="2.36e-9" izz="1.77e-4"/>
+        <origin rpy="0 0 0" xyz="0 0 0"/>
```
Comment (ahcorde) [inline mara_description/urdf/mara.urdf.xacro]: why you removed this inertias?

File: mara_description/urdf/mara.urdf.xacro [hunk 2]
```diff
@@ -280,39 +280,28 @@
...
+        <mass value="0.00001"/>
+        <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
       </inertial>
       <visual>
         <origin rpy="0 0 0" xyz="0 0 0"/>
         <geometry>
-          <mesh filename="package://mara_description/meshes/MARA_GripperAdapter.dae" scale="0.001 0.001 0.001"/>
+          <sphere radius="0"/>
```
Comment (ahcorde) [inline mara_description/urdf/mara.urdf.xacro]: sphere?

File: mara_description/urdf/mara_gazebo.urdf.xacro [hunk 3]
```diff
@@ -15,14 +15,11 @@
...
         <joint_name>motor6</joint_name>
-        <joint_name>left_inner_knuckle_joint</joint_name>
-        <joint_name>left_outer_knuckle_joint</joint_name>
-        <joint_name>left_outer_finger_joint</joint_name>
-        <joint_name>left_inner_finger_joint</joint_name>
-        <joint_name>right_inner_knuckle_joint</joint_name>
-        <joint_name>right_outer_knuckle_joint</joint_name>
-        <joint_name>right_outer_finger_joint</joint_name>
-        <joint_name>right_inner_finger_joint</joint_name>
+        <joint_name>robotiq_arg2f_base_to_left_inner_knuckle</joint_name>
```
Comment (ahcorde) [inline mara_description/urdf/mara_gazebo.urdf.xacro]: I think we are using mimic, I think we should use`joint_finger` instead all of these joints

File: mara_description/urdf/mara.urdf.xacro (hunk 1 above)
Comment (YueErro) [reply mara_description/urdf/mara.urdf.xacro]: This is just a dummy link.
This link was the gripper adapter. Which does not make sense to me and if we want to launch just MARA without gripper it should not have the gripper adapter i guess.

File: mara_description/urdf/mara.urdf.xacro (hunk 2 above)
Comment (YueErro) [reply mara_description/urdf/mara.urdf.xacro]: Same as above, we can put whatever.

File: mara_description/urdf/mara_gazebo.urdf.xacro (hunk 3 above)
Comment (YueErro) [reply mara_description/urdf/mara_gazebo.urdf.xacro]: I followed the old structure. I can replace all of them with joint_finger and test it.

Comment (YueErro) [reply mara_description/urdf/mara_gazebo.urdf.xacro]: You are right, in MARA repo is already like that, just with the `joint_finger`.

**Pull request merged**